guidelines.rst

```diff
@@ -164,6 +151,42 @@ because it makes heavy use of TensorFlow,
 which uses orange in its logo
 and other branding material.
 
-.. todo:: Be explicit about opacity values.
-          This will have to be played around with
-          once we have a project with its own branding.
+
+Color selection
+----------------
+
+First choose a primary color for the project,
+for example Nengo DL uses #ff6600.
+We must then select the tints of the primary color to create the
+gradation in the tuning curves. We **do not** use opacity or alpha
+values to create the gradation. Instead we use an online `color picking tool <https://www.tutorialrepublic.com/html-reference/html-color-picker.php>`_
```

Comment:
I'm a little worried what happens if this color picker disappears or changes, then all of a sudden none of our style guide instructions make sense anymore.  It'd be nice if we could define the gradient in absolute terms.  For example, we could define them as varying degrees of alpha, but convert those RGBA values to RGB (since we know the background colour).  There are automated tools for that as well, e.g. http://marcodiiga.github.io/rgba-to-rgb-conversion.

Reply:
Yea this is a good idea, I will update the colours and the method.

Should we link to a tool (like you've done) or just say to use a tool to convert RGBa -> RGB

Reply:
Yeah that'd actually be pretty sweet if we just had a little script that you input the base colour and it spits out all your colours for each logo.